CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.10)
+
+set(VCPKG_FEATURE_FLAGS manifests)
+
+project(openvpn-gui C)
+
+add_executable(${PROJECT_NAME} WIN32
+    access.c
+    echo.c
+    env_set.c
+    localization.c
+    main.c
+    manage.c
+    misc.c
+    openvpn.c
+    openvpn_config.c
+    options.c
+    passphrase.c
+    proxy.c
+    registry.c
+    save_pass.c
+    scripts.c
+    service.c
+    tray.c
+    viewlog.c
+    res/openvpn-gui-res.rc)
+
+configure_file(config-msvc.h config.h COPYONLY)
```

Comment:
@lstipakov  Currently PACKAGE_VERSION_RESOURCE is hard coded in config-msvc.h. For MSVC build how to extract it from configure.ac during build time?

Reply:
Good point. That's how: https://github.com/OpenVPN/openvpn-gui/pull/427